tests/support_server/login.pm

```diff
@@ -6,28 +6,43 @@
 
 use strict;
 use warnings;
-use base 'basetest';
 use base 'opensusebasetest';
 use testapi;
 use serial_terminal 'select_serial_terminal';
 use utils;
-use version_utils qw(is_desktop_installed is_tumbleweed);
+use version_utils qw(is_desktop_installed is_tumbleweed is_bootloader_grub2);
 
 sub run {
     my ($self) = @_;
     my $timeout = (is_tumbleweed) ? 180 : 80;
+
+    $self->override_default_bootloader unless is_bootloader_grub2;
```

Comment:
The `unless is_bootloader_grub2` condition means that you can just replace this with

```
my $prev_bootloader = get_var('BOOTLOADER');
set_var('BOOTLOADER', 'grub2');
[ ... ]
set_var('BOOTLOADER', $prev_bootloader);
```

Reply:
I'd prefer to keep it as it is atm, but your suggestion made me discover a small mistake on my end.